### ISSUE TITLE
Button doesn't display in component

### ISSUE THREAD

**Issue by adrobson** [angular/angular#54838]
### Which @angular/* package(s) are the source of the bug?

common, Don't known / other

### Is this a regression?

Yes

### Description

I am following the instructions for the housing tutorial.
I got to the end of lesson 3 but there are important differences between what I see in the browser and what I should be seeing. 

1. The button that is supposed to be in the home component doesn't show at all
2. There is supposed to be a border around the input box but that also doesn't show.

The same result in edge, firefox and chrome
![angular_screen](https://github.com/angular/angular/assets/10349413/266ca9cb-195c-43f4-ab17-3d05e81e9c46)

### Please provide a link to a minimal reproduction of the bug

Install versions as below. Follow instructions to the end of part 3.

### Please provide the exception or error you saw

```true
1. The button that is supposed to be in the home component doesn't show at all
2. There is supposed to be a border around the input box but that also doesn't show.
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Node v20.11.0
Angular cli 17.2.3
Schematics/Angular 17.2.3
Typescript 5.3.3
```

**Comment by adrobson** [comment]
It turns out this is to do with the styles.css file. In the examples they start with a styles.css which has values. Part of this file sets a PrimaryColor value. If there isn't a value for PrimaryColor the button is white on a white background ie invisible. 

However, I can't find anything in the instructions to populate this file. If you are creating a new project the file is empty and so the button is invisible. 

Therefore you might want to add something to populate this file. Otherwise there will be many like me (for a bit), who think Angular is broken.

**Comment by garrettld** [comment]
It sounds like you started a brand new project instead of starting by downloading the "default" app like the tutorial suggests. The default app includes the `styles.css` file with the `--primary-color` value already defined for you (and more).

However, the download functionality appears to be incomplete ever since commit https://github.com/angular/angular/commit/c4de4e1f894001d8f80b70297c5e576f2d11ec6f because the download no longer includes the necessary config files (tsconfig.ts, package.json, angular.json, etc.). If these _were_ included, the `styles.css` file would already be configured as the global stylesheet for you, as shown here in the now-deleted file: https://github.com/angular/angular/blob/8d58595478f0e9fcadbbd9ff21375e6875a9ef37/adev/src/content/tutorials/common/angular.json#L51

Kinda related: #53283, which started as a separate issue but the recent comments are talking about the missing config files.

I'm not sure if the solution is to partially undo https://github.com/angular/angular/commit/c4de4e1f894001d8f80b70297c5e576f2d11ec6f by restoring the `common` folder or if there's some other way to get these files now.

If the `common` folder that contains all the common config files is not restored, then we should at least update the README that still references it: https://github.com/angular/angular/blob/main/adev/src/content/tutorials/README.md#common

**Comment by adrobson** [comment]
Yes that's what happened. Maybe its just me that went wrong but I doubt it. Since the styles file is the only difference between starting a new project and downloading the other project, you could possibly add a note for those, like I did who are starting from their own project.

**Comment by madrizr** [comment]
Hello! I want to try to solve the issue

**Comment by Shamina05** [comment]
Hey, I'm new here. Let me know if anyone is currently looking at this isse?

**Comment by Bullettime23** [comment]
Hi everyone!
I figured out that the problem with this button is that it can't load its style variable properly. Add a global css variable --primary-color to your style.css file, and it'll do the job.
I hope google dev team will also add all the variables required in goo.gle/homes-app-start.
GL HF )

```
:root {
    --primary-color: blue;
}
```

![Image](https://github.com/user-attachments/assets/3f61e8c5-ad40-4ba0-ae17-365b62462c73)

**Comment by tysoncung** [comment]
I'm interested in working on this issue. Could you provide more context about what you're looking for? Any additional details about requirements or constraints would be helpful.

**Comment by Sanketh166** [comment]
Hi I'm new to open source and would like to work on this issue. Could you please assign it to me?

**Comment by arpitshukla9** [comment]
Hi @adrobson , I would like to work on this issue. I plan to update the documentation to include the missing --primary-color variable and a note for new project setup. Could you please assign this to me?

**Comment by axue3** [comment]
Hey! Just made a PR that attempts to address this issue. Happy to make the fixes to get it merged!
